### ISSUE TITLE
use standard style

### ISSUE THREAD

**Issue by mmckegg** [mmckegg/loop-drop-app#140]
It's all really inconsistent right now, but a lot of the new code is already using [standard style](https://github.com/feross/standard).

**Comment by mmckegg** [comment]
https://github.com/mattdesl/standardize

**Comment by ltmp** [comment]
I'm interested in trying

```
standard --format filename.js
```

And seeing if I can get away with doing that, maybe i'll run it on a branch or something.

**Comment by mmckegg** [comment]
@ltmp: that does handle a lot of cases, but not all unfortunately.

**Comment by ltmp** [comment]
@mmckegg yeah i'll give it a go and see how far it gets then do the rest by hand.
